doc/source/news.rst

```diff
@@ -3,6 +3,88 @@
 News
 ====
 
+.. _release-11-07:
+
+Release 11.07 - 2021-09-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/install/ubuntu`] Added support for MySQL 8.0 on Ubuntu 20.04 (Focal Fossa).
+
+  * There are below restrictions in the MySQL8 package.
```

Comment:
MySQL8 -> MySQL 8.0

Reply:
Thank you for your comment.
I've fixed it at 69120550abb2f49de486eb39dce1fb6d71f6e8f4.